### DISCUSSION TITLE
Update to 2.3.270 not working

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#11469] [Unsupported Versions] [answered]
Hi!

Why is soup telling me, that my 2.3.260 distributed deployment is up to date (You are already running the latest version of Security Onion.)? Isn't there 2.3.270 available?

Cheers, Ben

**Comment by xfaith** [comment, marked as answer]
@ben-sec  I maybe wrong, but Looking over the securityonion docs it looks like .270 will be available 10/6.  (Didnt see any other mentions of it ready/or announced on their blog.

**Comment by ben-sec** [comment]
I saw only the release notes in the 2.3 documentation and did not notice the release date. Thanks!
